CHANGELOG.md

```diff
@@ -9,12 +9,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [`x.y.z`] - Unreleased
 
+### Breaking changes:
+- Unreal Engine version `4.23` is no longer supported. We recommend upgrading to the newest version (`4.25.1`) to continue receiving updates.
+- When upgrading to Unreal Engine 4.25 you must:
+  1. In the engine folder, run `git fetch && git checkout 4.25-SpatialOSUnrealGDK-0.11.0`
+  1. Download and install the -v16 clang-9.0.1-based toolchain from [this Unreal Engine Documentation page](https://docs.unrealengine.com/en-US/Platforms/Linux/GettingStarted/index.html).
+  1. Run `Setup.bat`, which is located in the root directory of the UnrealEngine repository.
```

Comment:
I believe `InstallGDK.bat` runs these batch files for you now. Might be worth looking into.

Reply:
Checked now, don't think this is the case yet (we generateProjectFiles, but not Setup.bat)